.github/workflows/linux_unit_tests_with_latest_deps.yml

```diff
@@ -66,11 +66,16 @@ jobs:
           ! pip freeze | grep -E "xgboost|catboost|lightgbm|plotly|ipywidgets|category_encoders"
           exit $?
       - if: ${{ !matrix.core_dependencies }}
-        name: Installing Dependencies
+        name: Installing Dependencies and Prophet
```

Comment:
Right now we're installing our dependencies (which installs prophet since it's in our requirements file) and then we have to reinstall prophet with the right backend.

Although that works, I think we can simplify it a bit. I tried this out locally, and it seems to work (in the sense the unit tests pass)

```
pip install cmdstanpy==0.9.68
<compile cmdstan>
export CMDSTAN=<path-to-cmdstan>
export STAN_BACKEND=CMDSTANPY
pip install -r dev-requirements.txt
```

I think this also makes it easier to communicate to users how to properly install prophet with the right backend.

This reminds me, we should add a section to the install page in the docs about prophet installation! I'm hoping it can be this:

```
pip install cmdstanpy==0.9.68
<compile cmdstan>
export CMDSTAN=<path-to-cmdstan>
export STAN_BACKEND=CMDSTANPY
pip install evalml
```

Reply:
Good call, hopefully we should be able to remove this once we can get the cmdstan-ext wheel up for an easy pip install

Reply:
Update on this: we're not including Prophet in requirements because it's being installed as an extra-requirement in `setup.py`.  Therefore we'll be including the entire installation process for `Prophet` for `git-test-automl` and `git-test-other`.